internal/sidecarlogresults/sidecarlogresults.go

```diff
@@ -17,15 +17,26 @@ limitations under the License.
 package sidecarlogresults
 
 import (
+	"bufio"
+	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"os"
 	"path/filepath"
 
+	"github.com/tektoncd/pipeline/pkg/apis/config"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 	"golang.org/x/sync/errgroup"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
 )
 
+// ErrorReasonMaxResultSizeExceeded indicates that the result exceeded its maximum allowed size
+var ErrorReasonMaxResultSizeExceeded = fmt.Errorf("%s", "MaxResultSizeExceeded")
```

Comment:
1) Go style prefers brief names, and convention is to end errors with `Err`: `SizeExceededErr`.
2) Give the user helpful details -- like the max size: `SizeExceededErr  = fmt.Errorf("results size >%d limit", maxResultLimit)`

Reply:
Thanks! I updated the message to "result size > configured limit". I cannot quantify it because it can change based on a value in the configMap.

Reply:
Hmm.
I got a linter error in the build tests:
```
=========================================
==== TESTING IF GOLINT HAS BEEN DONE ====
=========================================
internal/sidecarlogresults/sidecarlogresults.go:38:5: error-naming: error var ResultSizeExceededErr should have name of the form ErrFoo (revive)
var ResultSizeExceededErr = fmt.Errorf("%s", "results size > configured limit")
    ^
==============================
==== GO LINT TESTS FAILED ====
==============================
```

Seems like I need to rename it to `ErrResultSizeExceeded` to follow `ErrFoo` convention.

Reply:
Oops, looks like I got the convention backwards, it's `ErrFoo` (not `FooErr`) -- sorry about that!

But I still recommend brevity: `ErrSizeExceeded`.